news/2 Fixes/6118.md

```diff
@@ -0,0 +1 @@
+Support the new renderer API in jupyter extension.
```

Comment:
I believe these need to be added to the CHANGELOG.

Reply:
100% correct thanks for the catch. Just forgot that. Will update.